addons/website/models/theme_models.py

```diff
@@ -222,13 +222,6 @@ class Theme(models.AbstractModel):
         'website.template_header_hamburger',
         'website.template_header_vertical',
         'website.template_header_sidebar',
-        'website.template_header_slogan',
-        'website.template_header_contact',
-        'website.template_header_boxed',
-        'website.template_header_centered_logo',
-        'website.template_header_image',
```

Comment:
I'll be testing migration at some point: what can I expect with this? Do users are reverted back to default header correctly if they choose one of those / customized stuff inside? :pray:

Reply:
I think the best behavior would be to keep the old header selected by the user and if he replaces it with a new one, he wouldn't be able to retrieve the old one.
So does this mean we should keep the old templates? :/

Reply:
> So does this mean we should keep the old templates? :/

If we wanted to do that, kinda yes. We could potentially keep the forks users made but that would still mean having to keep the related CSS and the related mentions in the options XML. So we won't do that. Being reverted to the default one or ideally the closest new one is what should happen (thanks to the migration script). We can do it later.